### PR TITLE
Remove orders

### PR DESCRIPTION
**Pull request by eliseydah**
Confirmation composition was created, debounce was added.

### CODE REVIEW

File: frontend/src/compositions/useConfirmations.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      acceptLabel: "Löschen",
+      accept: acceptCallback,
+    });
+  };
+  const showDeleteClientConfirmation = (acceptCallback: () => void) => {
+    confirm.require({
+      message: "Sind Sie sich sicher, dass der Kunde gelöscht werden soll?",
+      header: "Bestätigung",
+      rejectLabel: "Abbrechen",
+      rejectClass: "bg-transparent border text-red-500 border border-red-500 hover:bg-red-300/10",
```
Comment (fr-ser) [inline frontend/src/compositions/useConfirmations.ts:21]: Should we use the same class for the article deletion?

File: frontend/src/compositions/useConfirmations.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      acceptLabel: "Löschen",
+      accept: acceptCallback,
+    });
+  };
+
+  const showDeleteArticleConfirmation = (acceptCallback: () => void) => {
+    confirm.require({
+      message: "Wollen Sie den Artikel sicher wirklich löschen?",
```
Comment (fr-ser) [inline frontend/src/compositions/useConfirmations.ts:29]: For consistency:

```suggestion
      message: "Sind Sie sich sicher, dass der Artikel gelöscht werden soll?",
```

File: frontend/src/compositions/useNotifications.ts [hunk 3]
```diff
@@ -54,7 +54,20 @@ export default function useNotifications() {
...
+    toast.add({
+      severity: "info",
+      summary: "Der Auftrag wurde gelöscht",
+      life: 3000,
+    });
+  };
+  const showUpdateOrderNotification = () => {
+    toast.add({
+      severity: "info",
+      summary: "Der Auftrag wurden gespeichert",
```
Comment (fr-ser) [inline frontend/src/compositions/useNotifications.ts:67]:
```suggestion
      summary: "Der Auftrag wurde gespeichert",
```

**Comment by fr-ser** [comment]
I see this error message in the console. Do you have an idea what this might be about?

Steps to reproduce:
- go to this URL: http://localhost:5173/order
- hard refresh page
- click on "Aufträge" in navigation

See this warning
> Blocked aria-hidden on a <a> element because the element that just received focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden. <a href=​"/​order" tabindex=​"-1" aria-hidden=​"true" class=​"relative flex items-center py-3 px-5 leading-none select-none cursor-pointer no-underline overflow-hidden" data-pc-section=​"action">​…​</a>​flex

image

File: frontend/src/views/OrderView.vue [hunk 4]
```diff
@@ -26,7 +28,7 @@ let orderInfo = ref<OrderUpdate | OrderCreate>({
   discount_duration: 7,
   can_have_cash_discount: false,
 });
-
+const id = "id";
```
Comment (fr-ser) [inline frontend/src/views/OrderView.vue:31]: Should this be removed?

**Comment by eliseydah** [comment]
> I see this error message in the console. Do you have an idea what this might be about?
> 
> Steps to reproduce:
> 
> * go to this URL: http://localhost:5173/order
> * hard refresh page
> * click on "Aufträge" in navigation
> 
> See this warning
> 
> > Blocked aria-hidden on a  element because the element that just received focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden. <a href=​"/​order" tabindex=​"-1" aria-hidden=​"true" class=​"relative flex items-center py-3 px-5 leading-none select-none cursor-pointer no-underline overflow-hidden" data-pc-section=​"action">​…​​flex
> 
> image

I can't reproduce it, most likely it is caused by component library

**Pull request merged**